### PR TITLE
P2-2106-AB_job_outer_api

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -44,7 +44,7 @@
     <PackageVersion Include="Moq" Version="4.20.70" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="NLog.Web.AspNetCore" Version="5.3.8" />
-    <PackageVersion Include="NUnit" Version="4.0.0" />
```
Comment (ChrisJWoodcock) [inline Directory.Packages.props:47]: Red alert - this changes the package for every outer API - I expect you didn't mean to do that!

File: src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln [hunk 2]
```diff
@@ -19,6 +19,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{2648E357
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.ApprenticeFeedback.MockApis", "SFA.DAS.ApprenticeFeedback.MockApis\SFA.DAS.ApprenticeFeedback.MockApis.csproj", "{56EEE787-5656-44A8-AD60-F6D1AD491B2D}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth", "..\Shared\SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth\SFA.DAS.SharedOuterApi.Apprentice.GovUK.Auth.csproj", "{3922E528-51C0-4FD2-9BCB-8753F0F60202}"
```
Comment (ChrisJWoodcock) [inline src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln:22]: This is curious - why should this have been added here - was your branch synced with master when you started?

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: Directory.Packages.props (hunk 1 above)
Comment (karen-hanna) [reply Directory.Packages.props:47]: reverted to original

File: src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln (hunk 2 above)
Comment (karen-hanna) [reply src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln:22]: i was getting compilation errors and it implied this project needed to be included?

Comment (karen-hanna) [reply src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.sln:22]: looks like this is not required.  i think it was related to my dev environment

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**